src/explore-education-statistics-common/src/modules/methodology/components/MethodologyHelpAndSupportSection.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react';
+import ContactUsSection from '@common/modules/find-statistics/components/ContactUsSection';
+import { PublicationSummary } from '@common/services/publicationService';
+import { MethodologyPublication } from '@common/services/methodologyService';
+
+interface Props {
+  owningPublication: PublicationSummary | MethodologyPublication;
+}
+
+const MethodologyHelpAndSupportSection: FC<Props> = ({ owningPublication }) => {
+  return (
+    <>
+      <h2
+        className="govuk-!-margin-top-9"
+        data-testid="extra-information"
+        id="help-and-support"
+      >
+        Help and support
+      </h2>
+
+      <ContactUsSection
+        publicationContact={owningPublication.contact}
+        publicationTitle={owningPublication.title}
+      />
```

Comment:
Thinking about it, do we actually need this component at all? I'm not sure what value this is actually providing as it's just a heading and the contact section.

This is probably over-abstracted and I don't think it actually makes sense in the context it's being used in some cases (see other comment).

Reply:
I don't think it does any harm to have it as a component and it might be preferable because:
- it's consistent with release pages
- it'll help prevent future divergence between the admin and frontend versions as has happened a lot on releases

Not sure it really needs it's own test file though as it could be covered by MethodologyPage tests (in a more integration rather than unit testing style).

Reply:
I created this component for exactly the reason Amy mentioned above - that it's consistent with how the Release pages are structured. It also allows us to re-use the `ContactUsSection` component rather than creating a near-identical one for both releases and methodologies. 

Happy to remove the tests if you both want - I just think it's good practice to make sure every component has at the very least a basic "it renders" test.